core/src/main/java/org/pentaho/platform/scheduler2/action/SchedulerOutputPathResolver.java

```diff
@@ -192,19 +227,35 @@ protected boolean isValidOutputPath( @NonNull String path ) {
    */
   protected String getJobName() {
     return StringUtils.isNotBlank( getFilename() )
-      ? FilenameUtils.getPathNoEndSeparator( getFilename() ) // remove "/" + <jobName> + ".*" file pattern text
+      ? stripWildcardExtension( getFilename() )
```

Comment:
Previous code would cause the job name to always be empty, because file name never has any `/`.